src/plugins/intel_cpu/src/config.cpp

```diff
@@ -89,9 +89,19 @@ void Config::readProperties(const ov::AnyMap& prop, const ModelType modelType) {
                                    ". Expected only ov::Affinity::CORE/NUMA/HYBRID_AWARE.");
                 }
             }
+            if (key == ov::num_streams.name()) {
+                ov::Any value = val.as<std::string>();
+                auto streams_value = value.as<ov::streams::Num>();
```

Comment:
```suggestion
                auto streams_value = val.as<ov::streams::Num>();
```
Don't need cast to std::string, direct to `.as<ov::num_streams>()`

Reply:
If the type of input value is not int32, it is failed here. So I transform it to string first.